gnucash/gnucash-commands.cpp

```diff
@@ -70,60 +71,15 @@ scm_cleanup_and_exit_with_failure (QofSession *session)
         qof_session_destroy (session);
     }
     qof_event_resume();
-    gnc_shutdown (1);
+    return 1;
```

Comment:
Just as an aside, at some point we may want to change this to return a wider variation of error codes rather than just 0 or 1. This function will a good place to work that out. Otherwise not relevant to this PR :wink: